content/en/docs/chart_template_guide/function_list.md

```diff
@@ -214,26 +239,29 @@ print "Matt has " .Dogs " dogs"
 
 Types that are not strings are converted to strings where possible.
 
-Note, when two arguments next to each other are not strings a space is added between them.
+Note, when two arguments next to each other are not strings a space is added
+between them.
 
 ### println
 
 Works the same way as [print](#print) but adds a new line at the end.
 
 ### printf
 
-Returns a string based on a formatting string and the arguments to pass to it in order.
+Returns a string based on a formatting string and the arguments to pass to it in
+order.
 
 ```
 printf "%s has %d dogs." .Name .NumberDogs
 ```
 
-The placeholder to use depends on the type for the argument being passed in. This includes:
+The placeholder to use depends on the type for the argument being passed in.
+This includes:
 
 General purpose:
 
-* `%v` the value in a default format
-	when printing dicts, the plus flag (%+v) adds field names
+* `%v` the value in a default format when printing dicts, the plus flag (%+v)
+  adds field names
```

Comment:
It almost looks like the original formatting was intended to be meaningful here (although with the way the website displays, your change does not change the display).